opencti-platform/opencti-front/src/private/components/profile/notifications/NotificationsToolBar.tsx

```diff
@@ -193,6 +193,7 @@ const NotificationsToolBar: FunctionComponent<NotificationsToolBarProps> = ({
           },
         },
         onCompleted: () => {
+          handleClearSelectedElements();
```

Comment:
why do we need to add this call here since it's already called in `onSubmitCompleted()` (l162) ?

Reply:
Indeed, it's fixed. Thanks.